### DISCUSSION TITLE
m2strm v3.0.2.6

### DISCUSSION THREAD

**Discussion by trix7777** [trix7777/m2strm#17] [General]
- Fixed a long standing bug: string `longDate` was formatted in 12-hour format which in some cases caused duplicates of `newLogFile`/`purgeLogFile`.
- Corrected string name `userM3u8FileTemp` (was `userM3uFileTemp`).
- Keeping 4K and put `[UHD]` tag on such material. In Kodi and Jellyfin for instance, this will lead to Kodi showing duplicates where there are two qualities of the same movie; there is no way of telling them apart without playing them. Suggestions? [Discussions](https://github.com/trix7777/m2strm/discussions)
- Rudimentary fix for "Faulty downloaded M3U8 causes empty OutDirectory #7" reported by perryodk: When using this program to download the M3U8-file, validity will be performed by checking for `#EXTM3U` in the downloaded file before processing. If fail; processing stops.
- Other minor corrections and trimmings of N1 namings.

<hr /><em>This discussion was created from the release <a href='https://github.com/trix7777/m2strm/releases/tag/v3.0.2.6'>m2strm</a>.</em>

**Comment by Jimbo8169** [comment]
Seems to be Broken for the last few days keep getting error logs

25/10/2024 00:24:27: ERROR: The remote server returned an error: (404) Not Found.
25/10/2024 00:24:27: ERROR:    at System.Net.WebClient.DownloadFile(Uri address, String fileName)
   at m2strm.Program.Main(String[] args)

just wondering if there was a fix for this ?

**Reply by trix7777** [reply]
> Seems to be Broken for the last few days keep getting error logs
> 
> 25/10/2024 00:24:27: ERROR: The remote server returned an error: (404) Not Found.
> 25/10/2024 00:24:27: ERROR:    at System.Net.WebClient.DownloadFile(Uri address, String fileName)
>    at m2strm.Program.Main(String[] args)
> 
> just wondering if there was a fix for this ?

You give me almost no information to help you. 404 not found means just that, it can't be found. Maybe your iptv provider has recently changed something, you need to ask them.

**Reply by Jimbo8169** [reply]
sorry that was all the info the error log has every time i run the .exe it opens then intently closes with that error code thought some web service on the apps end might have been the cause as i can still download the m3u file no problems from my iptv provider. it has been working for months flawlessly until a few days ago. as i say i thought the app used a third party service to download the m3u some how and that that was down. im just a guy trying to work things out thanks for your help though

**Reply by Jimbo8169** [reply]
iptv provider says m3u8 play lists are banned in the uk so thats why im getting the error 404 but using a vpn does not seem to help me so either there m3u8 link is down but not m3u link or something is blocking m2strm.exe from getting the m3u8 link? prob a stupid question but can i get m2strm.exe to work with m3u instaed ? i know im out of my depth but i need answers :)
